frontend/public/module/k8s/k8s-models.ts

```diff
@@ -26,8 +26,12 @@ export const modelsToMap = (models: K8sKind[]): ImmutableMap<K8sResourceKindRefe
  */
 let k8sModels = modelsToMap(_.values(staticModels));
 
-const hasModel = (model: K8sKind) =>
-  k8sModels.has(referenceForModel(model)) || k8sModels.has(model.kind);
+const hasModel = (model: K8sKind) => {
+  if (model.crd) {
+    return k8sModels.has(referenceForModel(model));
+  }
+  return k8sModels.has(model.kind);
+};
```

Comment:
I believe this breaks Networking Service or Knative Service. A similar issue occurred here: https://github.com/openshift/console/pull/3358#discussion_r347492374

I'm in the middle of looking to Pipeline issues atm, but I saw this and wanted to comment. I believe if you look at the Admin => Networking => Services page you'll see a Tech Preview badge (as it's pulling the Knative Service). I'll load up this PR later if my assumptions are incorrect and see if I can find the related area.

Reply:
Thanks @andrewballantyne , I poked around in a lot of places looking for any issues this may cause but couldn't find any.

Reply:
@andrewballantyne you are correct, it pulls up the wrong services 👎

Reply:
In order to solve the issue I was originally dealing with when I created https://jira.coreos.com/browse/ODC-2125, I directly referred to the Models rather than allowing it to fetch it from the `allModels()` method. Is there anyway you can do that to achieve your goal on Topology?

Reply:
For instance, Knative Routing was done explicitly rather than allowing underlying pages to "lookup" the model via the `allModels()` map.

REF: https://github.com/openshift/console/pull/2917/files#diff-6f36dd2f169bcf981721e72cdd1e221dR8

Reply:
Thanks @andrewballantyne. I've updated the code to get the models from knative models rather than from k8s-models

Reply:
Awesome, I'll make some time for a review in a few hours. In the middle of playing with Topology for nefarious uses :)